velox/docs/functions/spark/math.rst

```diff
@@ -78,22 +78,22 @@ Mathematical Functions
     Returns ``x`` rounded up to the nearest integer.
     Supported types are: BIGINT, DOUBLE and DECIMAL.
 
-.. function:: checked_add(x, y) -> [same as x]
+.. function:: checked_add(x, y) -> [same as x] (ANSI compliant)
```

Comment:
Would you update PR description to explain why it is valid to add this without making any changes to the functions?

CC: @rui-mo